<launch>

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <param name="controller_frequency" value="3.0" /> 
    <rosparam file="$(find lili_navi)/param_lite/costmap_common_params_lite.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find lili_navi)/param_lite/costmap_common_params_lite.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find lili_navi)/param_lite/local_costmap_params_lite.yaml" command="load" />
    <rosparam file="$(find lili_navi)/param_lite/global_costmap_params_lite.yaml" command="load" />
    <rosparam file="$(find lili_navi)/param_lite/base_local_planner_params_lite.yaml" command="load" />
  </node>

</launch>
